docs/DeveloperGuide.md

```diff
@@ -436,121 +441,107 @@ All get commands are implemented in the following steps:
 4. The model is then updated such that the *filtered* list only displays patients whose details match the query
    arguments of that prefix
 
+#### Appointment Date (/appton)
+
+Getting the list of patients with an appointment on the query date involves the following steps:
+1. Prefix `/appton` is matched using an instance of `GetCommandParser`
+2. A new `GetAppointmentByDateCommandParser` instance is created and parses the user input (specifically the date inputted)
+3. A `GetAppointmentByDateCommand` instance containing the date of the appointment is created and returned
+4. The `GetAppointmentByDateCommand` is executed, accessing the list of `PastAppointment` of the specified patient
```

Comment:
This part sounds abit weird, like only 1 patient's past appointment is checked. 

Maybe can be like "... accesses the list of 'PastAppointment's for every patient, and filters for those who have a past appointment on the specified date."

Reply:
Fixed. Good catch!